src/components/molecules/Sidebar/Sidebar.tsx

```diff
@@ -47,6 +60,11 @@ const Sidebar = () => {
     isEnabled && setTab(TABS.LIVE_SCHEDULE);
   }, [isEnabled]);
 
+  const playAudio = useMemo(() => {
+    const audioObj = new Audio(notificationSound);
+    numberOfUnreadMessages && audioObj.play();
+  }, [numberOfUnreadMessages]);
```

Comment:
This is duplicated code. It would be good to extract this into a custom hook (as requested in previous reviews), rather than duplicating it in both locations:

- https://reactjs.org/docs/hooks-custom.html